src/MusicStore/project.json

```diff
@@ -51,14 +51,27 @@
   },
   "frameworks": {
     "net451": {},
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
       "dependencies": {
-        "NETStandard.Library": "1.5.0-*"
+        "Microsoft.NETCore.App": 
+        {
+          "version": "1.0.0-*",
+          "type": "platform"
+        }
       },
       "imports": [
         "dnxcore50",
```

Comment:
Not sure. Though if we are removing dnxcore50 imports, I would imagine this isn't the only place. Feel free to file an issue in Coherence-Signed.